### PR TITLE
Merge `next` into `master`

### PR DESCRIPTION
**Pull request by jsumners**
As titled.

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -26,19 +26,19 @@
...
-    "tsd": "^0.30.0",
+    "standard": "^17.1.0",
+    "tap": "^18.7.1",
+    "tsd": "^0.30.7",
     "yamljs": "^0.3.0"
   },
   "dependencies": {
-    "@fastify/accepts": "^4.0.0",
-    "fastify-plugin": "^4.0.0"
+    "@fastify/accepts": "^4.3.0",
```
Comment (jsumners) [inline package.json:40]: Blocking until this one is released.

File: package.json [hunk 2]
```diff
@@ -26,19 +26,19 @@
...
+    "fastify-plugin": "^4.5.1"
```
Comment (jsumners) [inline package.json:41]: Update.

File: package.json [hunk 3]
```diff
@@ -26,19 +26,19 @@
   },
   "homepage": "https://github.com/fastify/fastify-accepts-serializer#readme",
   "devDependencies": {
-    "@fastify/pre-commit": "^2.0.2",
-    "fastify": "^4.0.0-rc.2",
-    "msgpack5": "^6.0.0",
-    "protobufjs": "^7.0.0",
+    "@fastify/pre-commit": "^2.1.0",
+    "fastify": "^4.26.2",
```
Comment (jsumners) [inline package.json:30]: Update.

**Pull request merged**